Misc/NEWS.d/next/Library/2019-02-16-07-11-02.bpo-35899.cjfn5a.rst

```diff
@@ -0,0 +1 @@
+Enum has been fixed to correctly handle empty strings and strings with "weird" characters (ie. '!') without crashing. Original patch contributed by Maxwell. Assisted by Stéphane Wirtel.
```

Comment:
Please change the exclamation point and weird characters comment to an alphanumeric unicode comment.